src/content/writing/2019-10-09-on-goals-and-happiness.mdx

```diff
@@ -0,0 +1,98 @@
+
+#### A Page
+
+The particular idea of who I am goes back only ten or twelve years. Any further past and I have wide impressions: an easy ten-year-old desire to be, so absurdly generally, a _scientist_; the sharp sense of freedom while driving alone late at night with a new license<Footnote id="foot-1" href="#1">1</Footnote>. I can see the beginnings of myself, but the patterns and systems of my brain, my sense of being, do not feel coherently connected to who I am now. At eighteen I picked up the first thread of my personality that I can follow through to today -- when asked what I wanted to do, or to be, I would say that I only wanted to be happy. That was my goal; I am not goal oriented.
+
+I carried this goal, _happiness_, through school<Footnote id="foot-2" href="#2">2</Footnote> and along through the beginnings of my adulthood and career. I'll say at the outset, here, that happiness is a good direction and an awful goal. I'll say also that I was more earnest than smugly anti-establishment; I inherited a certain casual _everything works out_ attitude from my mom that I intellectualized to the point of being deeply unhelpful. It's true, too, that everything does work out, one way or another, and that I've been lucky. I come from a background of white- and male-privilege. I'm grateful for what I have -- what I've been given and what I've earned.
+
+Being happy, not letting things bother me, being _okay_, was a foundational part of my identity. This is aspirational and it's not entirely workable. In practice it looked like this: spending an enormous amount of mental energy to dismiss my feelings when things were _not okay_. It meant ignoring problems, rationalizing, and overindulging in patience. It meant, especially, avoiding emotional conflict.
+
+I did not often step into my life when it was hard and when I needed to. I wrapped discontent, sadness, and irritation in an intellectual mantra: I don't need to feel this. It will be okay. This was my process for existing; I have a habit of over-thinking and under-feeling<Footnote id="foot-3" href="#3">3</Footnote>. In my mid-twenties this slowly stopped working, and I descended into a semi-functional madness. My sense of identity was rapidly crumbling. I reached a breaking point, and I felt that I had broken. I did not wish to die, but in the morning I would pull the covers over my head, and hide, and think about not existing.
```

Comment:
I'm sorry you went through that. I'm glad you've made it through to the other-side, hopefully stronger and better off.